core/src/main/java/io/servicecomb/core/provider/consumer/InvokerUtils.java

```diff
@@ -72,18 +76,23 @@ public static Response innerSyncInvoke(Invocation invocation) {
           String.format("invoke failed, %s", invocation.getOperationMeta().getMicroserviceQualifiedName());
       LOGGER.debug(msg, e);
       return Response.createConsumerFail(e);
+    } finally {
+      invocation.triggerFinishedEvent();
     }
   }
 
   public static void reactiveInvoke(Invocation invocation, AsyncResponse asyncResp) {
     try {
+      triggerStartedEvent(invocation);
       ReactiveResponseExecutor respExecutor = new ReactiveResponseExecutor();
       invocation.setResponseExecutor(respExecutor);
 
       invocation.next(asyncResp);
     } catch (Throwable e) {
       LOGGER.error("invoke failed, {}", invocation.getOperationMeta().getMicroserviceQualifiedName());
       asyncResp.consumerFail(e);
+    } finally {
+      invocation.triggerFinishedEvent();
```

Comment:
seems we missed success /fail information

Reply:
wrapped asyncResp for triggerFinishedEvent in callback